docs/branch_tracking.md

```diff
@@ -0,0 +1,53 @@
+# Branch tracking
+
+This requires basic knowledge on how Jujutsu's branches work[^1].
+
+As of December 2023 Jujutsu tracks branches by default, which is confusing 
+users coming from Git and Mercurial. To smoothen the transition branch tracking
+was introduced. 
+
+## Tracking a branch
+
+To track a branch permanently use `jj branch track <name-of-branch>`. It will 
+now be available in your local repository until you untrack it or it is 
+deleted on the remote. 
+
+Example:
+
+```sh
+$ # List all available branches, as we want our colleague's branch.
+$ jj git branch list 
+$ # Find the branch.
+$ jj git branch track <name>
+$ # From this point on, branch <name> is tracked and will always be fetched.
+$ jj git fetch # Update the repository
+$ jj new <name> # Do some local testing, etc.
+```
+
+## Untracking a branch
+
+To no longer have a branch available in a repository, you can 
+`jj branch untrack` it. After that subsequent fetches will no longer copy the 
+branch into the local repository. 
+
+Example: 
+
+```sh
+$ # List all local and remote branches.
+$ jj branch list
+$ # Find the branch we no longer want to track.
+$ jj branch untrack <name>
+$ # From this point on, it won't be fetched anymore. 
+```
+
+
+## Ignored refs
```

Comment:
This feature isn't implemented (and I don't think we have detailed design yet.)

For the moment, we can use `git fetch --prune` and negative refspec in colocated repos. (I usually do that because libgit2 is super slow if the repo contained tons of jj/keep refs.)

Reply:
I removed this part.